server/tailwind.config.js

```diff
@@ -65,5 +65,22 @@ module.exports = {
       },
     },
   },
-  plugins: [require('@tailwindcss/line-clamp')],
+  plugins: [
+    require('@tailwindcss/line-clamp'),
+
+    // Add a custom plugin to support the `has` pseudo-classes.  For example: "has-checked:bg-blue-300".
```

Comment:
(question) Did you run into issues with the [builtin](https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-descendants) `has-*` support? It was added in v3.4.

Reply:
For some reason, the built-in support isn't working for me when I include it in Java, but I just found out that it works when I put it in `styles.css`, so I did that and removed the plugin.  I know it's not ideal that we're putting it in the stylesheet, but this is all going to get re-done with the North Star, so it's only temporary.

Reply:
That's so weird. I wonder if something is escaping the `[` and `]` characters on the java side.

Reply:
Hm, yeah, I wonder.  It would be nice to figure out, but I don't think it's worth the time since we're moving away from J2Html anyway.  Are you ok with me leaving it in the stylesheet?

Reply:
Yeah yeah. That's plenty fine

Reply:
Here's my problem. We might be moving away from j2html, but now I have a mystery that is going to consume me. lolol